Makefile

```diff
@@ -397,6 +397,11 @@ test-e2e-source:
 test-e2e-images:
 	ginkgo $(GINKGO_FLAGS) -focus="odo supported images e2e tests" tests/e2escenarios/
 
+# Run devfile e2e tests: odo devfile supported tests
+.PHONY: test-e2e-devfile
+test-e2e-devfile:
+    ginkgo $(GINKGO_FLAGS) -focus="odo devfile support tests" tests/e2escenarios/
```

Comment:
@rnapoles-rh this looks like 4 spaces - for makefile a tab is a tab. so please add a tab here instead of 4 spaces to solve the
`*** no separator`

Reply:
fixed